docs/satellite-configuration-tuning.rst

```diff
@@ -291,6 +291,27 @@ Dynflow Tuning
 
 Dynflow is the workflow management system and task orchestrator which is built as a plugin inside Foreman and is used to execute the different tasks of Satellite in an out-of-order execution manner. Under the conditions when there are a lot of clients checking in on Satellite and running a number of tasks, the Dynflow can take some help from an added tuning specifying how many executors can it launch.
 
+Increase sidekiq workers
+========================
+
+From Satellite 6.8, we have a new dynflow service named “dynflow-sidekiq”. It is recommended to increase the sidekiq workers in order to scale the foreman tasking system for bulk concurrent tasks like multiple CV publish/promote , capsule sync etc. There are two options available:
```

Comment:
Please add some generic description of what the service is - does below make sense @adamruzicka?

```suggestion
From Satellite 6.8, we have a new Dynflow service named “dynflow-sidekiq” that performs tasks scheduled by Dynflow. Sidekiq workers can be grouped into various queues to ensure lots of tasks of one type will not block execution of tasks of other type. Currently there are these ques:

* xxx - used to execute tasks yyy
* zzz - used to execute tasks aaa

It is recommended to increase the sidekiq workers in order to scale the foreman tasking system for bulk concurrent tasks like multiple CV publish/promote , capsule sync etc. There are two options available:
```

Reply:
looks good to me, except these two : 
* xxx - used to execute tasks yyy
* zzz - used to execute tasks aaa

Rest I added. Thank you.